src/components/benefactor/AddBenefactorForm.tsx

```diff
@@ -85,7 +76,14 @@ export default function AddBenefactorForm({ initialValues = defaults }: Benefact
     }
   }
   return (
-    <Grid container direction="column" component="section" className={classes.container}>
+    <Grid
+      container
+      direction="column"
+      component="section"
+      sx={{
+        maxWidth: '700px',
+        margin: '0 auto',
+      }}>
```

Comment:
You can give the grid a `maxWidth` prop which can be a predefined from the theme either `sm`,`md`,`lg` etc. We should generally try to use it instead of random pixel values

Reply:
Definitely no px values

Maybe the Grid component should not have any width at all but be wrapped in `<Container maxWidth="lg">` or similar.

However I've just migrated the styles, didn't tweak around